schema.sql

```diff
@@ -6,5 +6,8 @@ CREATE TABLE animals (
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN DEFAULT FALSE,
-    weight_kg NUMERIC
+    weight_kg NUMERIC,
```

Comment:
When executing this, it gave me an error: 
![Screenshot from 2023-06-14 00-00-02](https://github.com/CaptainBawa/Vet-Clinic/assets/93520178/2bbafd9e-abe9-4fc6-b2fd-53a0fb816507)
I suggest you remove the `,` on line 9.